packages/manager/modules/server-sidebar/src/template.html

```diff
@@ -1,36 +1,8 @@
-<div class="sidebar-menu position-absolute">
-    <div data-ng-if="$ctrl.isOpen" class="sidebar-menu_header d-flex p-4">
-        <div class="oui-navbar-menu__back h-100">
-            <button
-                type="button"
-                class="oui-button oui-button_link oui-button_icon-left"
-                data-ng-click="$ctrl.onClick()"
-            >
-                <span
-                    class="oui-icon oui-icon-chevron-left"
-                    aria-hidden="true"
-                ></span>
-                <span
-                    class="sr-only"
-                    data-translate="cloud_sidebar_public_cloud_project_toggle_close"
-                ></span>
-            </button>
-        </div>
-        <div class="w-100 text-truncate">
-            <h3
-                class="oui-navbar-menu__title text-white"
-                data-translate="{{:: 'server_sidebar_item_title_' + $ctrl.universe }}"
-            ></h3>
-        </div>
-    </div>
-    <div
-        data-sidebar-menu
-        data-ng-class="{
-            'nav-open': $ctrl.isOpen
-        }"
-        class="col-md-3 col-lg-2"
-    ></div>
-</div>
```

Comment:
I think this was meant to handle responsive navigation. Has it been moved to soemwhere else ?

Reply:
In fact, the way it was handled didn't work at all, and it was missing in Telecom.
This should have been managed directly form the main component.
So for now, I remove it to correct it later directly in `ng-ovh-sidebar-menu`.